js/rpg_managers/ImageManager.js

```diff
@@ -77,42 +85,42 @@ ImageManager.loadBitmap = function(folder, filename, hue, smooth) {
 };
 
 ImageManager.loadEmptyBitmap = function() {
-    var empty = this.cache.getItem('empty');
-    if (!empty) {
+    var empty = this._imageCache.get('empty');
+    if(!empty){
         empty = new Bitmap();
-        this.cache.setItem('empty', empty);
+        this._imageCache.add('empty', empty);
+        this._imageCache.reserve('empty', this._systemReservationId);
```

Comment:
Of course it won't happen, but if the added bitmap is purged immediately, this code will throw a TypeError.